test/client_server_functions.jl

```diff
@@ -134,7 +139,7 @@ function initiatingws(ws::WebSocket; msglengths = MSGLENGTHS, closebeforeexit =
 end
 
 test_serverws = WebSockets.ServerWS(
-    HTTP.RequestHandlerFunction(test_handler),
+    WebSockets.RequestHandlerFunction(test_handler),
```

Comment:
I like the change. Too many things were called prefixed HTTP. I would also think it's OK to export RequestHandlerFunction and similar, and refer to them wihout module prefix here in the tests.